### PR TITLE
Added status to outputs

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
Ready

## Description
Added the status to outputs

### CODE REVIEW

File: Integrations/AutofocusV2/AutofocusV2.py [hunk 1]
```diff
@@ -686,8 +687,8 @@ def top_tags_results_command():
         'ContentsFormat': formats['text'],
         'Contents': results,
         'EntryContext': {'AutoFocus.TopTagsResults(val.PublicTagName == obj.PublicTagName)': context,
-                         'AutoFocus.SessionsSearch(val.AFCookie == obj.AFCookie)': {'Status': status,
-                                                                                    'AFCookie': af_cookie}},
+                         'AutoFocus.TopTagsSearch(val.AFCookie == obj.AFCookie)': {'Status': status,
```
Comment (yuvalbenshalom) [inline Integrations/AutofocusV2/AutofocusV2.py]: why did you change the path?

Comment (ShahafBenYakir) [reply Integrations/AutofocusV2/AutofocusV2.py]: It was a mistake. The old path was only entered last week so no danger of breaking backward compatibility.
